fleet/lib/mimir/values.yaml

```diff
@@ -46,6 +46,8 @@ mimir:
       max_global_series_per_metric: 0
 
```

Comment:
It's possible to add `compactor_blocks_retention_period: 2y` here. This has the added benefit of being included in the config hash, which can be used to monitor config drift in a cluster.